ITGlueAPI/ITGlueAPI.psd1

```diff
@@ -18,7 +18,7 @@
     # -- MINOR version when you add functionality in a backwards-compatible manner, and
     # -- PATCH version when you make backwards-compatible bug fixes.
 
-    ModuleVersion = '2.2.0'
+    ModuleVersion = '2.1.0'
```

Comment:
The version has been bumped to 2.2.0 in upstream master, so don't forget to update your local branch to incorporate these changes.